cmd/globals.go

```diff
@@ -131,7 +131,7 @@ var (
 	globalHTTPStats = newHTTPStats()
 
 	// Time when object layer was initialized on start up.
-	globalBootTime time.Time
+	globalBootTime = time.Now().UTC()
```

Comment:
Why is the globalBootTime being changed in this PR? If we are defining globalBootTime as time when object layer was initialized, it is wrong to initialize it here.